deploy/template.yaml

```diff
@@ -919,80 +919,6 @@ Resources:
         TGPort: 8080
         VpcId: !Sub ${VpcStackName}-VpcId
 
-  # Metrics for frontend-vital-signs
```

Comment:
What about all the other bits around this https://github.com/govuk-one-login/ipv-cri-kbv-front/pull/1077 maybe a revert would work

Reply:
We are not reverting the changes, just removing the MetricFilter and that's it.